src/lua/api-gateway/validation/factory.lua

```diff
@@ -101,9 +102,9 @@ local function _generateHmacSignature()
     return hmacSignatureValidator:generateSignature()
 end
 
-local function _validateOAuthToken()
+local function _validateOAuthToken(obj)
     local oauthTokenValidator = OAuthTokenValidator:new()
-    return oauthTokenValidator:validateRequest()
+    BaseValidator:exitFn(oauthTokenValidator:validateRequest(obj))
```

Comment:
The composite validator idea sounds interesting. Do you have an example in mind that the [validatorsHandler](https://github.com/adobe-apiplatform/api-gateway-request-validation/blob/master/src/lua/api-gateway/validation/validatorsHandler.lua) wouldn't support ?